File: lab_2/Generator/generator.cpp
#include <iostream>
#include <random>

void get_pseudorandom_sequence(){
    /*
    *Generating a pseudorandom bit sequence
    */
    std::random_device rd;
    std::mt19937 engine;
    engine.seed(rd());

    std::uniform_int_distribution dist(0, 1);

    for (size_t index = 0; index < 128; index++){
        std::cout << dist(engine);
    }
}

int main(){
    get_pseudorandom_sequence();
}

File: lab_2/Generator/generator.java
import java.util.Random;

public class Main
{
    public static String GetRandomNumbers(int count) {
        /*
         * Generating a pseudorandom bit sequence
         */

        Random random = new Random();
        String random_sequence = "";

        for (int i = 0; i < count; i++) {
            random_sequence += random.nextInt(2);
        }
        return random_sequence;
    }

	public static void main(String[] args) {
		String random_sequence = GetRandomNumbers(128);
		System.out.print(random_sequence);
	}
}